1.8/administration/overlay-networks/index.md

```diff
@@ -41,9 +41,9 @@ The components of the overlay network interact in the following ways:
 
 Navstar maps IPs to names on your overlay network. During DNS lookup for a task’s IP, there are situations when it is unknown to the DNS server if the IP address stored in NetworkInfo is accessible or not. Three entries in Navstar DNS help to remedy this situation:
 
-* **taskname.marathon.agentip.dcos: agentip**: Provides the agent IP address.
-* **taskname.marathon.containerip.dcos: containerip**: Provides the container IP address.
-* **taskname.marathon.autoip.dcos: autoip**: Provides a best guess of a task's IP address.
+* **taskname.marathon.agentip.dcos.thisdcos.directory: agentip**: Provides the agent IP address.
+* **taskname.marathon.containerip.dcos.thisdcos.directory: containerip**: Provides the container IP address.
```

Comment:
The "<DNS address>: identifier:" notation here is confusing. It makes it seem like the ": identifier" part may be a part of the DNS address, which is misleading. Also, maybe it should be made clear that this is a resolvable DNS address? It's not made clear _how_ you're supposed to use this to obtain IP addresses, should have a mention that it's just DNS.